test/reality-capture.js

```diff
@@ -0,0 +1,78 @@
+const assert = require('assert');
+
+const { RealityCaptureClient } = require('..');
+
+describe('RealityCaptureClient', function() {
+    beforeEach(function() {
+        const { FORGE_CLIENT_ID, FORGE_CLIENT_SECRET } = process.env;
+        assert(FORGE_CLIENT_ID);
+        assert(FORGE_CLIENT_SECRET);
+        this.client = new RealityCaptureClient({ client_id: FORGE_CLIENT_ID, client_secret: FORGE_CLIENT_SECRET});
+        
+    });
+
+    describe('createPhotoScene()', function() {
+        before(function() {
+            const { SCENE_NAME, SCENE_CALLBACK, SCENE_FORMAT, SCENE_TYPE} = process.env;
+            assert(SCENE_NAME);
+            assert(SCENE_CALLBACK);
+            assert(SCENE_FORMAT);
+            assert(SCENE_TYPE);
+            this.scenename = SCENE_NAME;
+            this.scenecallback = SCENE_CALLBACK;
+            this.sceneformat = SCENE_FORMAT;
+            this.scenetype = SCENE_TYPE;
+        });
+        it('should create a new photoscene', async function() {
+            /* const photoscene = await this.client.createPhotoScene(this.scenename, this.scenecallback, this.sceneformat, this.scenetype);
+            assert(photoscene); */
+        });
+    });
+
+    describe('addImages()', function() {
+        before(function() {
+            const { SCENE_ID } = process.env;
+            assert(SCENE_ID);
+            this.sceneid = SCENE_ID;
```

Comment:
I'm not sure how the tests will behave if we're only setting the `this.sceneid` here. Other tests (e.g., `getPhotoSceneProgress` or `getPhotoScene`) depend on this property as well.

Reply:
Another thing is - these tests are run automatically by Travis CI with some hard-coded Forge client ID and secret, so ultimately I will have to create some ReCap scene with that ID and secret so that things work properly. So for now I'd say let's comment out the entire test suite for the Reality Capture APIs, and I will re-introduce the tests after the code is merged.